tba-key-setup.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
```

Comment:
Same as nix comment; not opposed to this, but I think it would be best to submit this as a separate PR rather than coupled with the teams page.